src/input/agent.rs

```diff
@@ -124,13 +126,31 @@ where
             }
         };
 
+        // Parse decision rule
+        let decision_rule = match agent_raw.decision_rule.as_str() {
```

Comment:
Elsewhere we've made things case insensitive, so:

```suggestion
        let decision_rule = match agent_raw.decision_rule.to_ascii_lowercase().as_str() {
```